src/relayserver/RelayServer.ts

```diff
@@ -153,10 +162,12 @@ export class RelayServer extends EventEmitter {
     }
     // Check that the fee is acceptable
     if (parseInt(req.relayRequest.relayData.pctRelayFee) < this.config.pctRelayFee) {
-      throw new Error(`Unacceptable pctRelayFee: ${req.relayRequest.relayData.pctRelayFee} relayServer's pctRelayFee: ${this.config.pctRelayFee}`)
```

Comment:
you repeat yourself. how about "checkRange(title,min,max,value)"

Reply:
Huh? That's not part of this PR

Reply:
I see it part of the diff.
you should create a PR against release, and then it will contain only its own content, and not other PRs, otherwise, I can't tell.

Reply:
The only change is indentation.